Assets/Mirror/Core/Tools/Extensions.cs

```diff
@@ -9,15 +9,29 @@ public static class Extensions
         public static string ToHexString(this ArraySegment<byte> segment) =>
             BitConverter.ToString(segment.Array, segment.Offset, segment.Count);
 
+        public static readonly Dictionary<string, int> StableHashes = new Dictionary<string, int>();
```

Comment:
```suggestion
        static readonly Dictionary<string, int> StableHashes = new Dictionary<string, int>();
```

Reply:
I made it public on purpose, so it could be iterated from user code without modding Mirror to debug stuff like this:
```
Unknown message id: 33978. This can happen if no handler was registered for this message.
NetworkClient: failed to unpack and invoke message. Disconnecting. 
```